### DISCUSSION TITLE
How to set up SSO in environment file

### DISCUSSION THREAD

**Discussion by jesseyu1984918** [inventree/InvenTree#8075] [Q&A]
I have gone through the documentation but still didn't get on how to enable SSO, can anyone help?
Single Sign On is not enabled for this server

**Comment by SchrodingersGat** [comment]
You need to first enable SSO login under [Login Settings](https://docs.inventree.org/en/stable/settings/global/#login-settings)

then, setup a SSO provider by following these instructions - https://docs.inventree.org/en/stable/settings/SSO/#

Setting up SSO is an advanced process, and requires some level of advanced knowledge
